public/javascripts/releases-controllers.js

```diff
@@ -241,6 +241,14 @@
             if ('undefined' === typeof(ticket.diffs)) {
                 ticket.diffs = [];
             }
+
+            // Remove empty requirements
+            for (var key in $scope.newDiff.requirements) {
+                if (!$scope.newDiff.requirements[key]) {
```

Comment:
I couldn't find any issues with this file with jshint. Maybe I did it wrong...
Changed it any way.